lib/tasks/db.rake

```diff
@@ -43,7 +43,14 @@ namespace :db do
     end
 
     system 'createdb', '-h', config.host, '-p', config.port, '-U', config.username, config.database
```

Comment:
No response, so I'm going to merge this in. @mhelmetag if you have time and space cycles, would you submit a fix for this?